src/EventTickets/Migrations/CreateEventTicketTypesTable.php

```diff
@@ -43,8 +44,9 @@ public function run() {
 			event_id INT UNSIGNED NOT NULL,
 			label TEXT NULL,
 			description TEXT NULL,
-			price INT UNSIGNED NULL,
-			max_available INT UNSIGNED NULL,
+			price DECIMAL(18,10) NOT NULL,
+			max_tickets_available INT UNSIGNED NULL,
```

Comment:
Why did you change the column name?

Reply:
Resolved: a5d894416d2d9e6ac8e594326ca2769773d3615d